packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -188,6 +198,12 @@ export const GiantTrackTile = ({
     source: FavoriteSource.TRACK_PAGE
   })
 
+  const { isEnabled: isRemixContestEnabled } = useFeatureFlag(
+    FeatureFlags.REMIX_CONTEST
+  )
+
+  const isRemixContest = isRemixContestEnabled && !isOwner
```

Comment:
will need to update this later to check for the events on the track when those updates are in